src/classes/issues-processor.ts

```diff
@@ -121,6 +121,11 @@ export class IssuesProcessor {
     }
 
     for (const issue of issues.values()) {
+      // Stop the processing if no more operations remains
+      if (!this.operations.hasRemainingOperations()) {
+        break;
```

Comment:
maybe a log here so the user knows why issues weren't processed?

Reply:
No need, this code will be executed after the break
https://github.com/C0ZEN/stale/blob/2f93a2a49f4c195ae653bb1d88195f1ada50c8f7/src/classes/issues-processor.ts#L407

Reply:
ah I see it now. It's just a big loop 😂

Reply:
I am not fond of this because it's not obvious but yeah, one day it would be nice to make the code cleaner!